tests/test_cmd.sh

```diff
@@ -37,23 +37,37 @@ ARE_IMAGES_EQUAL="${BINARY_DIR}/tests/are_images_equal"
 TMP_ENCODED_FILE=/tmp/encoded.avif
 DECODED_FILE=/tmp/decoded.png
 PNG_FILE=/tmp/kodim03.png
+JPG_FILE=/tmp/kodim03.jpg
 
 # Prepare some extra data.
 set +x
 echo "Generating a color PNG"
 "${AVIFENC}" -s 10 "${TESTDATA_DIR}"/kodim03_yuv420_8bpc.y4m -o "${TMP_ENCODED_FILE}" &> /dev/null
 "${AVIFDEC}" "${TMP_ENCODED_FILE}" "${PNG_FILE}"  &> /dev/null
+"${AVIFDEC}" "${TMP_ENCODED_FILE}" "${JPG_FILE}"  &> /dev/null
```

Comment:
Should the message at line 44 also mention JPEG now?

Reply:
This is now a C++ test.